README.md

```diff
@@ -115,6 +116,8 @@ We currently accept tokens on the following chains:
 - `optimism-sepolia`
 - `mode`
 - `pgn`
+- `lisk`
+- `lisk-sepolia-testnet`
```

Comment:
can the "-testnet" suffix be removed? It's implicit with it being sepolia

Reply:
Removed the `-testnet` suffix from all the places. Please re-review 🙏